api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -249,4 +250,35 @@
           "select distinct domain_id as domain, type, is_standard as standard from cb_criteria order by domain, type, is_standard",
       nativeQuery = true)
   List<DbMenuOption> findMenuOptions();
+
+  @Query(
+      value =
+          "select surveyId, version, itemCount from( "
+              + "select csv.survey_id as surveyId, csv.version as version, sum(csa.item_count) as itemCount, csv.display_order "
```

Comment:
Question: here we have sum(csa.item_count) vs just returning csa.item_count in method **findSurveyVersionByQuestionConceptIdAndAnswerConceptId** , 

Is it because  for findSurveyVersionByQuestionConceptId,  its aggregating the item_count of all answers for question_id passed?

Reply:
correct.